uint16 n_bin
float32 range
float32 vos
uint8 angle_step_size
uint16 left_limit
uint16 right_limit
float32 gain